### PR TITLE
feat(unlock-app) - collapse keys by network

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

https://user-images.githubusercontent.com/20865711/199325547-9555121a-c5df-496d-9ed0-3f103d0b01d8.mov

**Collapsed**

Screenshot 2022-11-01 at 20 46 35

**Not Collapsed**

Screenshot 2022-11-01 at 20 46 45

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by kalidiagne** [comment]
@ChialiTsai any suggestions to do it in a better way?

**Comment by ChialiTsai** [comment]
image
We have this UI component, expand/collapsable in the metadata config, wanna try use the same?
with arrow and full width as an entire row

**Comment by kalidiagne** [comment]
> image We have this UI component, expand/collapsable in the metadata config, wanna try use the same? with arrow and full width as an entire row

Super! thk, just updated with the same component 

Screenshot 2022-11-02 at 06 16 40

File: unlock-app/src/components/interface/locks/List/elements/LockList.tsx [hunk 1]
```diff
@@ -21,14 +26,31 @@ const LocksByNetwork = ({ network, owner }: LocksByNetworkProps) => {
...
+                {locks?.map((lock: Lock, index: number) => (
+                  <LockCard key={index} lock={lock} network={network} />
+                ))}
+                {loading && <LockCardPlaceholder />}
+              </div>
+            </Disclosure.Panel>
+          </div>
+        )}
+      </Disclosure>
     </div>
```
Comment (searchableguy) [inline unlock-app/src/components/interface/locks/List/elements/LockList.tsx:54]: Think this part is repeated a few times. Let's ask chiali on a universal collapse and use it everywhere?

**Review by searchableguy — APPROVED** [review]

**Pull request merged**